W-16026636 Making fixes (#341)
mulesoft/docs-access-management: modules/ROOT/pages/troubleshooting-anypoint-platform-access.adoc

Talk to your account representative or use https://www.mulesoft.com/contact[the MuleSoft contact form] to request an Enterprise license for your on-premises Mule Runtime instances or to create a subscription account on CloudHub.

=== Where do I sign in with external identity management?

> If your organization uses external identity management, sign into your account at *anypoint.mulesoft.com/accounts/login/_yourorganizationname_*.

Reviewer: ```suggestion
If your organization uses external identity management, sign in to your account at *anypoint.mulesoft.com/accounts/login/_yourorganizationname_*.
```

=== What account should I use?

> If you have one or more existing MuleSoft community accounts tied to the email address at which you received your invitation, MuleSoft prompts you to use your existing account to sign in and accept the invitation. You can also choose to create a new account by following the *Sign Up* link at the bottom of the form.

Reviewer: ```suggestion
If existing MuleSoft community accounts are associated with the email address where you received your invitation, MuleSoft prompts you to use your existing account to sign in and accept the invitation. You can also create a new account by clicking the *Sign Up* link at the bottom of the form.
```
